### DISCUSSION TITLE
When will Vercel seek HIPPA Compliance

### DISCUSSION THREAD

**Discussion by CalamariDude** [vercel/community#4272] [Feature Question]
### Question

Hi
We really want to host our services on Vercel but that won't be possible if they aren't HIPPA compliant. 
Jad

**Comment by amyegan** [comment]
Good question! There is no specific timeline on [HIPAA](https://www.hhs.gov/hipaa/for-professionals/privacy/laws-regulations/index.html) compliance yet, but we are looking at how we can make it happen 🙂

**Comment by harlantwood** [comment]
Lots more on this topic here: https://github.com/orgs/vercel/discussions/690
